clients/cli/src/main/java/org/apache/gravitino/cli/FullName.java

```diff
@@ -153,7 +153,7 @@ public String getNamePart(int position) {
       return names[position];
     }
 
-    System.err.println(ErrorMessages.MISSING_NAME);
+    //    System.err.println(ErrorMessages.MISSING_NAME);
```

Comment:
I'm unsure whether I should remove this code.

Reply:
It needs to be left in as that occurs when the name option missing